docs/developers/celestia-node.md

```diff
@@ -13,16 +13,17 @@ cd $HOME
 rm -rf celestia-node
 git clone https://github.com/celestiaorg/celestia-node.git
 cd celestia-node/
+git checkout tags/v0.3.0-rc1
 make install
 ```
 
 Verify that the binary is working and check the version with `celestia version` command:
 
 ```console
 $ celestia version
-Semantic version: v0.2.0
-Commit: 1fcf0c0bb5d5a4e18b51cf12440ce86a84cf7a72
-Build Date: Fri 04 Mar 2022 01:15:07 AM CET
+Semantic version: v0.3.0-rc1
+Commit: 5843c5e102651206fe11c685c163f267b141b103
+Build Date: Thu 26 May 2022 02:32:58 PM CEST
 System version: amd64/linux
-Golang version: go1.17.5
+Golang version: go1.18.2
```

Comment:
This isn't easily maintainable. A CI job should be added that runs the above steps, then runs `celestia version` and compares the output to these lines.

Reply:
Its true, if this changes a lot it isnt maintainable, but for now i think its still better than nothing while theres a better solution. 

We could also just mention that the user needs to check if his version matches the latest release

Reply:
#116